frontend/public/components/utils/file-input.tsx

```diff
@@ -172,7 +172,9 @@ const DroppableFileInputWithTranslation = withDragDropContext(
       this.state = {
         inputFileName: '',
         inputFileData: this.props.inputFileData || '',
-        inputFileIsBinary: this.props.inputFileIsBinary || isBinary(null, this.props.inputFileData),
+        inputFileIsBinary:
+          this.props.inputFileIsBinary ||
+          isBinary(null, Buffer.from(this.props.inputFileData, 'base64')),
```

Comment:
Do we know that the input file is base64 encoded all the time?
```suggestion
          isBinary(null, Buffer.from(this.props.inputFileData)),
```

Reply:
This was the change previously, however when the data is base64-encoded, `isBinary` will erroneously say that the file is text